spec/components/layout_for_public_spec.rb

```diff
@@ -47,6 +47,18 @@ def component_name
     assert_select ".gem-c-layout-for-public .gem-c-layout-header", false
   end
 
+  it "can omit the feedback component" do
+    render_component(hide_feedback_form: true)
+
+    assert_select ".gem-c-layout-for-public .gem-c-feedback", false
+  end
+
+  it "can add a product name in the header" do
+    render_component(product_name: "Account")
+
+    assert_select ".gem-c-layout-for-public .gem-c-header__product-name", text: "Account"
+  end
```

Comment:
This is the same as a test for the header component - do you think it's needed for the public layout component as well?

https://github.com/alphagov/govuk_publishing_components/blob/1dee4ae94c9187455bd5b5cf714e4bf5c0ef219f/spec/components/layout_header_spec.rb#L46-L50

Reply:
Yeah I think my intention with this test is to verify that `product_name` is correctly passed through from `layout_for_public` to `layout_header`.